deployment/nginx/conf/nginx.conf

```diff
@@ -0,0 +1,50 @@
+events {}
+http {
+    server {
+        listen 80 default_server;
+        listen [::]:80 default_server;
+
+        location / {
+            return 301 https://$host$request_uri;
+        }
+    }
+    server {
+        listen 443 ssl http2;
+        listen [::]:443 ssl http2;
+
+        ssl_certificate /etc/letsencrypt/live/blue.bicikl-project.eu-0001/fullchain.pem;
+        ssl_certificate_key /etc/letsencrypt/live/blue.bicikl-project.eu-0001/privkey.pem;
+        ssl_session_timeout 1d;
+        ssl_session_cache shared:MozSSL:10m;
+        ssl_session_tickets off;
+
+        ssl_dhparam /etc/letsencrypt/dhparam;
+
+        # intermediate configuration
+        ssl_protocols TLSv1.2 TLSv1.3;
+        ssl_ciphers ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA>
+        ssl_prefer_server_ciphers off;
+
+        # HSTS (ngx_http_headers_module is required) (63072000 seconds)
+        add_header Strict-Transport-Security "max-age=63072000" always;
+
+        # OCSP stapling
+        ssl_stapling on;
+        ssl_stapling_verify on;
+
+        # verify chain of trust of OCSP response using Root CA and Intermediate certs
+        ssl_trusted_certificate /etc/letsencrypt/live/marketplace.cetaf.org-0001/fullchain.pem;
```

Comment:
Don't think this does much as this points to marketplace.cetaf

Reply:
True, temporarily changed it back to the bicikl one, have to change it when we do a real publish to the cetaf server